dotcom-rendering/src/layouts/CrosswordLayout.tsx

```diff
@@ -460,15 +464,13 @@ export const CrosswordLayout = (props: WebProps) => {
 						webUrl={article.webURL}
 						webTitle={article.webTitle}
 						showBottomSocialButtons={
-							article.showBottomSocialButtons &&
-							renderingTarget === 'Web'
+							article.showBottomSocialButtons
 						}
 					/>
 				</Section>
 				{renderAds && (
 					<Section
 						fullWidth={true}
-						data-print-layout="hide"
```

Comment:
I've removed the existing `data-print-layout` here as it doesn't get passed through by `Section` so has no effect.